drivers/test/src/commonMain/kotlin/com/squareup/sqldelight/driver/DriverTest.kt

```diff
@@ -61,40 +64,40 @@ abstract class DriverTest {
     val changes = database.getConnection().prepareStatement("SELECT changes()", SELECT, 0)
 
     query.executeQuery().use {
-      assertThat(it.next()).isFalse()
+      assertFalse(it.next())
     }
 
     insert.bindLong(1, 1)
     insert.bindString(2, "Alec")
     insert.execute()
-    assertThat(changes.executeQuery().apply { next() }.getLong(0)).isEqualTo(1)
+    assertEquals(changes.executeQuery().apply { next() }.getLong(0), 1)
```

Comment:
Flip arguments, expected comes first
```suggestion
    assertEquals(1, changes.executeQuery().apply { next() }.getLong(0))
```